app/Http/Controllers/Api/SignupsController.php

```diff
@@ -38,6 +35,22 @@ public function __construct(SignupService $signups, PostService $posts)
     {
         $this->signups = $signups;
         $this->posts = $posts;
+
+        $this->transformer = new SignupTransformer;
+    }
+
+    /**
+     * Returns signups.
+     * GET /signups
+     *
+     * @param Request $request
+     * @return ]Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+        $query = $this->newQuery(Signup::class);
+
+        return $this->paginatedCollection($query, $request);
```

Comment:
We might want to use a separate transformer for our v3 routes as well, since that would allow us to change the behavior of how this is displayed: for example, resetting default includes (only returning signups, unless `?include=posts` is provided) & changing response structure (`signup_id` → `id`, `signup_source` → `source`).

Reply:
ah that makes a lot of sense, thanks @DFurnes !